hw2/05-form.html

```diff
@@ -7,14 +7,131 @@
+                  style="width: 100%"
+                />
+                <label for="email" class="form-label" style="font-weight: bold"
+                  >Email:</label
+                >
+                <br />
+                <input
+                  autocomplete="email"
+                  type="email"
+                  class="form-control"
+                  name="email"
+                  id="email"
+                  style="width: 100%"
+                />
+              </div>
+              <label
+                for="registration"
+                class="form-label"
+                style="font-weight: bold"
+                >Registration Status:</label
+              >
+              <select
+                class="form-select"
+                aria-label="Registration Status"
+                name="registration"
+                id="registration"
+              >
+                <option selected>Choose an Option</option>
+                <option value="yes">Registered</option>
+                <option value="no">Not Registered</option>
+              </select>
+              <fieldset style="margin-top: 2em">
+                <legend style="font-size: 17px">
+                  <label for="questions" style="font-weight: bold"
+                    >Have you taken any of the following courses?</label
+                  >
+                  <div class="form-check">
+                    <input
+                      type="checkbox"
+                      class="form-check-input"
+                      id="pLang"
+                    />
+                    <label class="form-check-label" for="pLang"
+                      >Programming Languages</label
+                    >
+                  </div>
+                  <div class="form-check">
+                    <input type="checkbox" class="form-check-input" id="os" />
+                    <label class="form-check-label" for="os"
+                      >Operating Systems</label
+                    >
+                  </div>
+                  <div class="form-check">
+                    <input
+                      type="checkbox"
+                      class="form-check-input"
+                      id="full-stack"
+                    />
+                    <label class="form-check-label" for="full-stack"
+                      >Full Stack Web Development</label
+                    >
+                  </div>
+                </legend>
+              </fieldset>
+              <label
+                for="questions"
+                class="form-label"
+                style="font-weight: bold"
+                >Anything else?</label
+              >
+              <br />
+              <textarea
+                class="form-control"
+                name="questions"
+                id="questions"
+                cols="30"
+                rows="5"
+                style="width: 99%; height: 6em"
+              >
+              </textarea>
+              <button
```

Comment:
The button styles are not applying properly. Consider wrapping them in a ```<div>``` element and applying the styles there. That worked for me because I was having the same issue.